shared/oae/api/oae.api.util.js

```diff
@@ -175,24 +177,26 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
          * @api private
          */
         var init = function(callback) {
-            // List all macro files available in /shared/oae/macros to this list to have them
-            // available for JST templates
-            var macroFiles = ['header.html', 'list.html'];
-
-            // Load all macros using the `require` command
-            var macros = _.map(macroFiles, function(macroFile) {
-                return 'text!/shared/oae/macros/' + macroFile;
-            });
-            require(macros, function() {
+            // Note: These need to have their full URLs in the file in order for minification to
```

Comment:
It was just recently broken in the My Search PR as I hadn't run it on an optimized build yet: https://github.com/oaeproject/3akai-ux/pull/4005/files#diff-e129e3b2200876e4dd414cabe246bb1aR180

You'll be happy to know I've run Inviting Guests on an optimized build and we've bug bashed it a couple times :P